currency/storage.go

```diff
@@ -208,9 +228,22 @@ func (s *Storage) SetDefaultFiatCurrencies(c ...Code) error {
 	return nil
 }
 
+// SetStableCoins assigns the stable currency list and adds it to the running
+// list
+func (s *Storage) SetStableCoins(c Currencies) error {
+	for i := range c {
+		err := s.currencyCodes.UpdateCurrency("", c[i].String(), "", 0, Stable)
```

Comment:
This highlights that you shouldn't update things in `UpdateCurrency` unless they != 0 or != "". If those were set, you'd be overriding `fullName`, `blockchain` and `ID`  here